### PR TITLE
Added Roboto font and remove text shadow

### PR DESCRIPTION
**Pull request by tul1**
This PR resolves the following issues:
https://github.com/ekumenlabs/ekumenlabs.github.io/issues/159
https://github.com/ekumenlabs/ekumenlabs.github.io/issues/160

### CODE REVIEW

**Comment by tul1** [comment]
The result of adding Roboto font and removing the text shadows is the shown in the images beneath:
With the new changes:

image

Original:

image

@german-e-mas let know if the changes are acceptable to merge them.

File: source/stylesheets/main.scss [hunk 1]
```diff
@@ -47,7 +49,8 @@ h1, h2, h3, h4, h5, h6 {
   color: $purple;
   font-weight: 600;
   margin: 1em 0 .75em;
-  text-shadow: 0 1px 1px rgba(0, 0, 0, 0.4);
+//  text-shadow: 0 1px 1px rgba(0, 0, 0, 0.4);
```
Comment (german-e-mas) [inline source/stylesheets/main.scss]: Don't leave commented code. Remove this line.

File: source/stylesheets/main.scss [hunk 2]
```diff
@@ -47,7 +49,8 @@ h1, h2, h3, h4, h5, h6 {
...
+  text-shadow: none;
```
Comment (german-e-mas) [inline source/stylesheets/main.scss]: Is this needed, or can it be removed?

Comment (tul1) [reply source/stylesheets/main.scss]: Removing it
image

Leaving it:
image

There is a difference. So, we should keep it I think.

Comment (german-e-mas) [reply source/stylesheets/main.scss]: I'm not sure why the `carousel-caption` applies the `text-shadow` to the `h3` text of it.

I am fine with the shadow in text over images.

**Comment by gecofernando** [comment]
+1 to the removal of gray backgrounds.

I'm not against changing the current typeface (which one is it, by the way?) but I *personally* don't like how Roboto looks.

The drop shadow in this case is not needed -on top of a white background- but it does provide some depth and adds the contrast agains the background needed in some cases (like the example on top of an image).

File: source/stylesheets/main.scss [hunk 3]
```diff
@@ -4,6 +4,8 @@
  * Imports / Includes
  * ========================================================================== */
 @import url(http://fonts.googleapis.com/css?family=Open+Sans:400,600|Noto+Sans:400); // Google fonts
+@import url(http://fonts.googleapis.com/css?family=Roboto); // Roboto Font from Google
```
Comment (german-e-mas) [inline source/stylesheets/main.scss]: Another comment: Check if we can have the https version of the font.

**Comment by german-e-mas** [comment]
Thanks for the comments @gecofernando ! The font being used before is [Open Sans](https://fonts.google.com/specimen/Open+Sans).

I agree on keeping the shadow over an image.

**Comment by gecofernando** [comment]
I don't see the reason to this change. It's not awful but the way the sentences are being cut is awkward to read and restricting the text to a side is unnecessary (as it's not blocking more/less of anything in particular).

As a side note, the white text gets hard to read against the clearer part of the background.

**Comment by german-e-mas** [comment]
I wouldn't stress too much with the carousel. I do think the `text-shadow` in it needs to go back.

Have you checked that the `text-shadow` is missing on the other sections of the site as well?

**Comment by tul1** [comment]
@german-e-mas I've just found another text-shadow in the code, I replaced for its value for a none and tested in the website. Everything now it's looking good. would like to give a look ?

File: source/stylesheets/home.scss [hunk 4]
```diff
@@ -134,7 +134,7 @@
       word-break: break-word;
       z-index: 2;
       text-align: center;
-      text-shadow: 2px 2px 1px  $black;
+      text-shadow: none;
```
Comment (german-e-mas) [inline source/stylesheets/home.scss]: I wouldn't change this. This `text-shadow` is applied to the thumbnails' white text. It helps the text to stand out of the image.

File: source/stylesheets/main.scss (hunk 2 above)
Comment (german-e-mas) [reply source/stylesheets/main.scss]: I suggest not to use `text-shadow: none` here, only removing it.

The `text-shadow` of the carousel title comes from an internal bootstrap style. And that is good to have, as it helps the titles stand out of the image.

By setting `none`, we override this style, and we don't want that.

Comment (tul1) [reply source/stylesheets/main.scss]: I added `text-shadow` to the header. But I removed it from the rest of the text.

**Comment by tul1** [comment]
@german-e-mas I am ready for a new review.

File: source/stylesheets/home.scss [hunk 5]
```diff
@@ -7,6 +7,9 @@
 
 .home-header {
   position: relative;
+  h1, h2, h3, h4, h5, h6 {
```
Comment (german-e-mas) [inline source/stylesheets/home.scss]: The heading we use is `h3`. Target only that one.

**Pull request merged**